theme/lumo/vaadin-number-field-styles.html

```diff
@@ -0,0 +1,29 @@
+<link rel="import" href="../../../vaadin-lumo-styles/font-icons.html">
+<link rel="import" href="../../../vaadin-lumo-styles/sizing.html">
+<link rel="import" href="../../../vaadin-lumo-styles/mixins/field-button.html">
+
+<dom-module id="lumo-number-field" theme-for="vaadin-number-field">
+  <template>
+    <style include="lumo-field-button">
+      :host {
+        width: 8em;
+      }
+
+      :host([has-controls]) [part="value"] {
+        text-align: center;
```

Comment:
note: it makes sense to include most of these styles below (excluding font-size and padding) into the core styles, as they are currently copy-pasted in both Lumo and Material